incognia_test.go

```diff
@@ -160,6 +165,7 @@ var (
 		InstallationID: "installation-id",
 		AccountID:      "account-id",
 		ExternalID:     "external-id",
+		Eval:           &shouldNotEval,
```

Comment:
Why isn't the payment fixture split like the login's, i.e. `paymentFixture` and `paymentFixtureWithEval`? Should we have both here?